<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendFrame</class>
 <widget class="QFrame" name="SendFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1260</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Lato</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: 0px; background-color: #282d31; color: #ddd; border-radius: 5px;</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="m_send_frame_main_layout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="sendLabel">
       <property name="minimumSize">
        <size>
         <width>1200</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>55</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Lato</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #ddd; 
font-size: 20px; 
padding: 15px;
font-family: 'Lato';
border-bottom: 1px solid #444;</string>
       </property>
       <property name="text">
        <string>Send Funds</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="topMargin">
      <number>15</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>750</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>750</width>
         <height>35</height>
        </size>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
       <widget class="QPushButton" name="m_sendButton">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>180</width>
          <height>35</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#m_sendButton {color: orange; ; border: 1px solid orange; font-size: 14px; border-radius: 5px; font-family: 'Lato';}

QPushButton#m_sendButton:hover {color:#FFD700; ; border: 1px solid orange; font-size: 14px; border-radius: 5px;font-family: 'Lato';}</string>
        </property>
        <property name="text">
         <string>SEND FUNDS</string>
        </property>
       </widget>
       <widget class="QPushButton" name="m_backButton">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>0</y>
          <width>180</width>
          <height>35</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#m_backButton {color: #ccc; border: 1px solid #444; font-size: 14px; border-radius: 5px; font-family: 'Lato';}

QPushButton#m_backButton:hover {color: #fff; border: 1px solid #444; font-size: 14px; border-radius: 5px; font-family: 'Lato';}</string>
        </property>
        <property name="text">
         <string>BACK</string>
        </property>
       </widget>
       <widget class="QPushButton" name="m_addressBookButton">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>0</y>
          <width>180</width>
          <height>35</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#m_addressBookButton {color: #ccc; border: 1px solid #444; font-size: 14px;  border-radius: 5px;font-family: 'Lato';}

QPushButton#m_addressBookButton:hover {color: #fff; border: 1px solid #444; font-size: 14px; border-radius: 5px; font-family: 'Lato';}</string>
        </property>
        <property name="text">
         <string>ADDRESS BOOK</string>
        </property>
       </widget>
       <widget class="QPushButton" name="m_clearAllButton">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>0</y>
          <width>180</width>
          <height>35</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>180</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#m_clearAllButton {color: #ccc; border: 1px solid #444; font-size: 14px; border-radius: 5px;font-family: 'Lato';}

QPushButton#m_clearAllButton:hover {color: #fff; border: 1px solid #444; font-size: 14px; border-radius: 5px; font-family: 'Lato';}</string>
        </property>
        <property name="text">
         <string>CLEAR ALL</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>750</width>
         <height>571</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>750</width>
         <height>520</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="pidLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>500</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #ddd; font-size: 14px; font-family: 'Lato';</string>
        </property>
        <property name="text">
         <string>Payment ID (Optional) - Required at some Exchanges</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="m_paymentIdEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>740</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 3px; border: 1px solid #444; border-radius: 5px; font-family: 'Lato'; font-size: 14px; color: #aaa;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="adressLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>500</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #ddd; font-size: 14px; font-family: 'Lato';</string>
        </property>
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="m_addressEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>740</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 3px; border: 1px solid #444; border-radius: 5px; font-family: 'Lato'; font-size: 14px; color: #aaa;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLineEdit" name="m_labelEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>360</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>360</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>360</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 3px; border: 1px solid #444; border-radius: 5px; font-family: 'Lato'; font-size: 14px; color: #aaa;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="labelLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>281</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #ddd; font-size: 14px; font-family: 'Lato';</string>
        </property>
        <property name="text">
         <string>Label </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="m_amountEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>360</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>360</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>360</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 3px; border: 1px solid #444; border-radius: 5px; font-family: 'Lato'; font-size: 14px; color: #aaa;</string>
        </property>
        <property name="text">
         <string>0.000000</string>
        </property>
       </widget>
       <widget class="QLabel" name="amountLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>291</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #ddd; font-size: 14px; font-family: 'Lato';</string>
        </property>
        <property name="text">
         <string>Amount</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="m_messageEdit">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>190</y>
          <width>360</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>360</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>360</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 3px; border: 1px solid #444; border-radius: 5px; font-family: 'Lato'; font-size: 14px; color: #aaa;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="amountLabel_2">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>160</y>
          <width>301</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #ddd; font-size: 14px; font-family: 'Lato';</string>
        </property>
        <property name="text">
         <string>Message (Optional)</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="m_feeSpin">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>270</y>
          <width>360</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>360</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>360</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 3px; border: 1px solid #444; border-radius: 5px; font-family: 'Lato'; font-size: 14px; color: #aaa;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>0.000100000000000</double>
        </property>
        <property name="maximum">
         <double>100000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000010000000000</double>
        </property>
        <property name="value">
         <double>0.000100000000000</double>
        </property>
       </widget>
       <widget class="QLabel" name="feeLabel">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>240</y>
          <width>291</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #ddd; font-size: 14px; font-family: 'Lato';</string>
        </property>
        <property name="text">
         <string>Fee</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="availableLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #ddd; font-size: 14px; font-family: 'Lato';</string>
        </property>
        <property name="text">
         <string>Available Balance</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="m_balanceLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>360</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>360</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>360</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 3px; border: 1px solid #444; border-radius: 5px; font-family: 'Lato'; font-size: 14px; color: #aaa;</string>
        </property>
        <property name="text">
         <string notr="true">0.000000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="m_nodeFee">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>350</y>
          <width>360</width>
          <height>31</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>360</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>360</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 3px; border: 1px solid #444; border-radius: 5px; font-family: 'Lato'; font-size: 14px; color: #aaa;</string>
        </property>
        <property name="text">
         <string notr="true">0.001000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="nodeFeeLabel">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>320</y>
          <width>291</width>
          <height>25</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lato</family>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #ddd; font-size: 14px; font-family: 'Lato';</string>
        </property>
        <property name="text">
         <string>Node Fee</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>400</y>
          <width>741</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding: 3px; font-size: 16px; color: #aaa;</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:16px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Lato'; font-size:14px;&quot;&gt;Please ensure that the Address and the Payment ID (if included) are correct before sending. Funds sent to the wrong address will be lost.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_clearAllButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFrame</receiver>
   <slot>clearAllClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_sendButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFrame</receiver>
   <slot>sendClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_backButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFrame</receiver>
   <slot>backClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1198</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_addressBookButton</sender>
   <signal>clicked()</signal>
   <receiver>SendFrame</receiver>
   <slot>addressBookClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1090</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_messageEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SendFrame</receiver>
   <slot>updateFee()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>625</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>sendClicked()</slot>
  <slot>backClicked()</slot>
  <slot>clearAllClicked()</slot>
  <slot>updateFee()</slot>
  <slot>addressBookClicked()</slot>
  <slot>mixinValueChanged(int)</slot>
 </slots>
</ui>
